### PR TITLE
Adding lights controller and test activity.

### PR DESCRIPTION
**Pull request by veniosg**

### CODE REVIEW

File: app/src/main/java/de/culo/lucyo/lights/LucyoLightController.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    public static void turnOn(Context c) {
+        startWithAction(c, ACTION_TURN_ON);
+    }
+
+    public static void turnOff(Context c) {
+        startWithAction(c, ACTION_TURN_OFF);
+    }
+
+    private static void startWithAction(Context c, String action) {
```
Comment (frapontillo) [inline app/src/main/java/de/culo/lucyo/lights/LucyoLightController.java:51]: 👍

File: app/src/main/java/de/culo/lucyo/lights/WiFiBox.java [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+                        "The group number must be between 1 and 4");
+        }
+
+        // adjust color
+        messages[1] = padMessage(COMMAND_COLOR, value);
+
+        // send messages
+        sendMultipleMessages(messages, DEFAULT_SLEEP_BETWEEN_MESSAGES);
+    }
+}
```
Comment (frapontillo) [inline app/src/main/java/de/culo/lucyo/lights/WiFiBox.java:712]: 712 LoC like a boss

**Review by frapontillo — APPROVED** [review]
fuck it

**Comment by alexstyl** [comment]
ecd6dcfa-8259-11e6-8705-97803f5c90aa

File: app/src/main/java/de/culo/lucyo/lights/WiFiBox.java (hunk 2 above)
Comment (veniosg) [reply app/src/main/java/de/culo/lucyo/lights/WiFiBox.java:712]: Long productive day today

**Pull request merged**